packages/electron/README.md

```diff
@@ -0,0 +1,504 @@
+    - [BacktraceClient Options](#backtraceclient)
+    - [Manually send an error](#manually-send-an-error)
+    - [Modify/skip error reports](#modifyskip-error-reports)
+    - [SDK Method Overrides](#sdk-method-overrides)
+
+## Basic Integration
+
+### Install the package
+
+```
+$ npm install @backtrace/electron
+```
+
+### Integrate the SDK in the main process
+
+Add the following code to your application before all other scripts to report Electron errors to Backtrace.
+
+```ts
+// Import the BacktraceClient from @backtrace/electron with your favorite package manager.
+import { BacktraceClient, BacktraceConfiguration } from '@backtrace/electron';
+
+// Configure client options
+const options: BacktraceConfiguration = {
+    // Submission url
+    // <universe> is the subdomain of your Backtrace instance (<universe>.backtrace.io)
+    // <token> can be found in Project Settings/Submission tokens
+    url: 'https://submit.backtrace.io/<universe>/<token>/json',
+};
+
+// Initialize the client with the options
+const client = BacktraceClient.initialize(options);
+
+// By default, Backtrace will send an error for Uncaught Exceptions and Unhandled Promise Rejections
+
+// Manually send an error
+client.send(new Error('Something broke!'));
+```
+
+### Integrate the SDK in the renderer process
+
+You can use any browser based Backtrace SDK in your renderer process, such as `@backtrace/browser` or
+`@backtrace/react`. Make sure to install it aside `@backtrace/electron`:
+
+```
+$ npm install @backtrace/react
+```
+
+To add Backtrace to the renderer process with Electron IPC support, you need to use a
+[preload script](https://www.electronjs.org/docs/latest/tutorial/tutorial-preload) in your `BrowserWindow`. Once you
+have one, add this import to it:
+
+```ts
+import '@backtrace/electron/main/preload';
+```
+
+This will ensure that IPC APIs that Backtrace uses are loaded correctly.
+
+After that, you can initialize Backtrace in your renderer process. There are two things that differ from the normal SDK
+initialization:
+
+1. URL is not used - you can leave it as an empty string,
+1. You need to use `addBacktraceElectron` with the builder.
+
+For example, if you are initializing the SDK using the `initialize` function:
+
+```ts
+const client = BacktraceClient.initialize(
+    {
+        url: '',
+        name: 'My Renderer App',
+        version: '0.0.1',
+    },
+    (builder) => addBacktraceElectron(builder),
+);
+```
+
+Or, if you are using the `build` function:
+
+```ts
+const builder = BacktraceClient.builder({
+    url: '',
+    name: 'My Renderer App',
+    version: '0.0.1',
+});
+
+addBacktraceElectron(builder);
+
+const client = builder.build();
+```
+
+#### Synchronized initialization
+
+When adding Electron to the renderer SDK, data is exchanged with the main process to ensure same session IDs are used.
+By default, this is done asynchronously as soon as possible. This may however be not enough in case of startup crashes
```

Comment:
every error will go thru the main process anyway. So the application session shouldn't be different, am I correct? Also crashes always go thru the main process so the application.session is always there.

Reply:
There are other things potentially using the session ID.

Reply:
Metrics use `sessionId`, and have a public property exposing the session ID for instance.

I feel like I answered this before... github is losing my comments!